R/execution.r

```diff
@@ -133,6 +134,14 @@ execute = function(request) {
     assign('quit', quit, envir = .GlobalEnv)
     assign('q',    quit, envir = .GlobalEnv)
     
+    send_error_msg <- function(msg){
```

Comment:
I don't think it makes sense to put that anywhere else: if the `silent` is outside the function, it's simple a `send_response` call and then I could simple use that...